src/auto-evo/AutoEvo.cs

```diff
@@ -9,7 +9,7 @@ public static AutoEvoRun CreateRun(GameWorld world)
         {
             var result = new AutoEvoRun(world);
 
-            if (Settings.Instance.RunAutoEvoDuringGamePlay)
+            if (Settings.Instance.RunAutoEvoDuringGamePlay.Value)
```

Comment:
I just looked up implicit conversion operators in C#, and those seem to exist. Would it be possible to use that here to implicitly convert the options objects to their internal value to not have to put `.Value` everywhere?